core/src/main/java/org/apache/spark/memory/TaskMemoryManager.java

```diff
@@ -390,7 +390,7 @@ public MemoryBlock allocatePage(long size, MemoryConsumer consumer) {
     try {
       page = memoryManager.tungstenMemoryAllocator().allocate(acquired);
     } catch (OutOfMemoryError e) {
-      logger.warn("Failed to allocate a page ({} bytes), try again.",
+      logger.warn("Failed to allocate a page ({} bytes), try again.", e,
```

Comment:
Could you add the new error message to the PR description too, @AngersZhuuuu ?

You can compare it via `**BEFORE**` and `**AFTER**`.

Reply:
How about current? @dongjoon-hyun